src/main/steps/c100-rebuild/applicant/address/common/address-select.ts

```diff
@@ -36,7 +36,7 @@ const cy = content => {
     {
       attributes: { id: 'totalAddressesFound' },
       value: -1,
-      text: `${addresses.length} address${addresses?.length !== 1 ? 'es' : ''} found - welsh`,
+      text: `Daethpwyd o hyd i ${addresses.length} cyfeiriad`,
```

Comment:
Hi @sealDebolina , there is one condition to check the number of addresses found . If it is 1, then need to display other text and if its multiple, then it should display other text. Hope that is covered as part of this check in welsh?

Reply:
as per communication with Welsh team Plural Check is not required with this translation.we have got following translation from them:
<img width="996" alt="Screenshot 2022-12-22 at 14 37 01" src="https://user-images.githubusercontent.com/116593651/209157644-9e944789-0523-4ff0-931e-ceaf269d2e98.png">